data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -7,23 +7,26 @@ IRONIC_IMAGE=$(image_for ironic)
 IPA_DOWNLOADER_IMAGE=$(image_for ironic-ipa-downloader)
 COREOS_DOWNLOADER_IMAGE=$(image_for ironic-machine-os-downloader || image_for ironic-rhcos-downloader)
 
-# This image is templated in via the installer pkg/asset/ignition/bootstrap/bootstrap.go
+# This image is templated in via the installer pkg/asset/ignition/bootstrap/common.go
 RHCOS_BOOT_IMAGE_URL="{{.BootImage}}"
+RHCOS_LIVE_IMAGE_URL="{{.PlatformData.BareMetal.ClusterOSImage}}"
 
 # This DHCP range is used by dnsmasq to serve DHCP to the cluster. If empty
 # dnsmasq will only serve TFTP, and DHCP will be disabled.
 DHCP_RANGE="{{.PlatformData.BareMetal.ProvisioningDHCPRange}}"
 
 # Used by ironic to allow ssh to running IPA instances
-IRONIC_RAMDISK_SSH_KEY="{{.SSHKey}}"
+IRONIC_RAMDISK_SSH_KEY="$(echo -n "{{.SSHKey}}")"
 
 # First we stop any previously started containers, because ExecStop only runs when the ExecStart process
 # e.g this script is still running, but we exit if *any* of the containers exits unexpectedly
-for name in ironic-api ironic-conductor ironic-inspector ironic-ramdisk-logs dnsmasq httpd mariadb ipa-downloader coreos-downloader; do
+for name in ironic-api ironic-conductor ironic-inspector ironic-ramdisk-logs dnsmasq httpd mariadb ipa-downloader coreos-downloader image-customization; do
```

Comment:
Can we remove ipa-downloader and the related `IPA_DOWNLOADER_IMAGE` above?

Reply:
yup